presto-docs/src/main/sphinx/release/release-0.281.rst

```diff
@@ -0,0 +1,58 @@
+=============
+Release 0.281
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix timeout for optimization rules.
+* Improve memory accounting for rehashing.
+* Improve outer join where the join key is from the output of another outer join which can have many null values. The optimization is controlled by session property ``randomize_outer_join_null_key_strategy``.
+* Add ability to express concurrency levels as a function of number of available workers.
```

Comment:
Should we mention the property that controls this?

Reply:
They have provided additional release notes in the "Resource Groups Changes" section with property name.  Line# 27.

Reply:
Let's please just mention this in the resource group changes section and remove from here.